packages/ast-spec/tests/fixtures-with-differences-errors.shot

```diff
@@ -49,6 +56,23 @@ exports[`AST Fixtures List fixtures with Error differences 1`] = `
   "TSESTree errored but Babel didn't": [
     "declaration/ExportAllDeclaration/fixtures/_error_/named-non-identifier/fixture.ts",
     "declaration/ExportNamedDeclaration/fixtures/_error_/aliased-literal/fixture.ts",
+    "declaration/VariableDeclaration/fixtures/_error_/const-id-definite-init/fixture.ts",
+    "declaration/VariableDeclaration/fixtures/_error_/const-id-definite-type-init/fixture.ts",
+    "declaration/VariableDeclaration/fixtures/_error_/declare-const-id-definite-init/fixture.ts",
+    "declaration/VariableDeclaration/fixtures/_error_/declare-const-id-definite-no-init/fixture.ts",
+    "declaration/VariableDeclaration/fixtures/_error_/declare-const-id-definite-type-no-init/fixture.ts",
+    "declaration/VariableDeclaration/fixtures/_error_/declare-let-id-definite-no-init/fixture.ts",
+    "declaration/VariableDeclaration/fixtures/_error_/declare-let-id-definite-type-no-init/fixture.ts",
+    "declaration/VariableDeclaration/fixtures/_error_/declare-var-id-definite-no-init/fixture.ts",
+    "declaration/VariableDeclaration/fixtures/_error_/declare-var-id-definite-type-no-init/fixture.ts",
+    "declaration/VariableDeclaration/fixtures/_error_/let-id-definite-init/fixture.ts",
+    "declaration/VariableDeclaration/fixtures/_error_/let-id-definite-no-init/fixture.ts",
+    "declaration/VariableDeclaration/fixtures/_error_/let-id-definite-type-init/fixture.ts",
+    "declaration/VariableDeclaration/fixtures/_error_/using-id-definite-init/fixture.ts",
+    "declaration/VariableDeclaration/fixtures/_error_/using-id-definite-type-init/fixture.ts",
+    "declaration/VariableDeclaration/fixtures/_error_/var-id-definite-init/fixture.ts",
+    "declaration/VariableDeclaration/fixtures/_error_/var-id-definite-no-init/fixture.ts",
+    "declaration/VariableDeclaration/fixtures/_error_/var-id-definite-type-init/fixture.ts",
```

Comment:
Babel doesn't detect whether a definite assignment is put on a non-declared let/var with type and no init. This is arguably a Babel bug